src/overlays/actors/ovl_En_Ma_Yto/z_en_ma_yto.h

```diff
@@ -2,14 +2,72 @@
 #define Z_EN_MA_YTO_H
 
 #include <global.h>
+#include "overlays/actors/ovl_En_Ma_Yts/z_en_ma_yts.h"
 
 struct EnMaYto;
 
+typedef void (*EnMaYtoActionFunc)(struct EnMaYto*, GlobalContext*);
+
+// Limbs of OBJECT_MA2
+typedef enum {
```

Comment:
```suggestion

typedef enum {
```
Is this comment needed?

Reply:
Not really, I think I'll remove it